src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -148,7 +128,6 @@
     grid-template-columns: var(--pf-c-about-modal-box--lg--grid-template-columns);
     grid-template-rows: max-content max-content auto;
     max-width: var(--pf-c-about-modal-box--lg--MaxWidth);
```

Comment:
```suggestion
```

looks like you can remove this, it's currently breaking the `max-width` on lg screens. Although I think this begs the question of whether the about modal should manage its width at all, or if it should just consume the available space and be controlled by the modal. I'd lean toward the latter - wdyt @srambach @mceledonia @mcarrano?

fwiw if you fix this and let the `max-width` apply, it looks like this in a modal with the modal width set to 50% of the viewport.

<img width="2898" alt="Screen Shot 2022-11-09 at 12 24 05 PM" src="https://user-images.githubusercontent.com/35148959/200910855-44044a0c-b5df-4c59-8e4f-6a0b299a1017.png">

Reply:
Not sure I completely understand the question @mcoker . What's there now looks good to me. The model should have a max-width IMO. Maybe I'm struggling to differentiate between Modal and About modal.

Reply:
@mcoker updated

Reply:
@mcarrano sorry, it's definitely confusing. The _about modal_ has a custom width and height set, since it is it's own implementation of a modal. With this update, I was proposing we remove the _about modal_'s width, and just let it fill the size of whatever container it's in, which would mean the _modal_ component is used to manage the _about modal_'s width. The _modal_ has width variations for small, medium, large, and a "custom" width.